resources/views/database/modifySource.blade.php

```diff
@@ -0,0 +1,70 @@
+                <div class="card h-100 shadow  mx-auto text-center">
+                    <div class="card-header">
+                        <h3>Current Node</h3>
+                        <h4>{{$source->node_name}}</h4>
+                    </div>
+                    <div class="card-body">
+                        <div class="d-flex">
+                            <div class="input-group mb-3">
+                                <div class="input-group-prepend">
+                                    <span class="input-group-text" id="inputGroup-sizing-default">New Name</span>
+                                </div>
+                                <input type="text" name="newSourceValue" class="form-control" aria-label="Default"
+                                       aria-describedby="inputGroup-sizing-default" value="{{$source->node_name}}">
+                                <input type="text" name="node_id" value="{{$source->node_id}}" hidden>
+                            </div>
+                        </div>
+                        <div class="d-flex">
+                            <div class="input-group mb-3">
+                                <div class="form-check form-check-inline">
+                                    @if($source->is_source === true)
+                                        <input class="form-check-input" id="is_source_checkbox" type="checkbox" name="is_source" checked>
+                                    @else
+                                        <input class="form-check-input" id="is_source_checkbox" type="checkbox" name="is_source">
+                                    @endif
+                                    <label class="form-check-label" for="is_source_checkbox">Source</label>
+                                </div>
+                                <div class="form-check form-check-inline">
+                                    @if($source->is_destination === true)
+                                        <input class="form-check-input" id="is_destination_checkbox" type="checkbox" name="is_destination" checked>
+                                    @else
+                                        <input class="form-check-input" id="is_destination_checkbox" type="checkbox" name="is_destination">
+                                    @endif
+                                    <label class="form-check-label" for="is_destination_checkbox">Destination</label>
+                                </div>
+                                <div class="form-check form-check-inline">
+                                    @if($source->is_fixture === true)
+                                        <input class="form-check-input" id="is_fixture_checkbox" type="checkbox" name="is_fixture" checked>
+                                    @else
+                                        <input class="form-check-input" id="is_fixture_checkbox" type="checkbox" name="is_fixture">
+                                    @endif
```

Comment:
Looks good overall, but could we put the @if @endif just around the "checked" attribute?
So it would look like: 
`<input class="form-check-input" id="is_destination_checkbox" type="checkbox" name="is_destination" @if($source->is_destination === true) checked @endif>`

Reply:
true true ok